CMakeLists.txt

```diff
@@ -231,12 +228,17 @@ if (WIN32)
         /wd4800 # disable warning type' : forcing value to bool 'true' or 'false' (performance warning)
         /wd4503 # identifier' : decorated name length exceeded, name was truncated
     )
+    if(NOT BUILD_SHARED_LIBS)
```

Comment:
This is not right. Whether you link the runtime library statically or dynamically is independent of whether you build your library as static or dynamic. You can have all four combinations.

Reply:
What's what I meant minimal fix. LOL. I addressed it in the next diff.